frontend/src/components/VideoQualityLabel/VideoQualityLabel.tsx

```diff
@@ -0,0 +1,25 @@
+
+import type { VideoQualityLabelProps } from './VideoQualityLabel.types';
+import styles from './VideoQualityLabel.module.css';
+import dolbyVector from '../../assets/images/dolbyVector.svg';
+
+
+const VideoQualityLabel = ({ type, className }: VideoQualityLabelProps) => {
+  if (type === 'DolbyVision') {
```

Comment:
What do ypu think about making it a bit shorter? Like this:
=> type === 'DolbyVision' ? (dolbyVisionIcon) : (textLabel);